### PR TITLE
[fmt] transformations

### PR DESCRIPTION
**Pull request by RMeli**
Format `MDAnalysis/transformations` and `MDAnalysisTests/transformations`. 

## Developers certificate of origin
- [x] I certify that this contribution is covered by the LGPLv2.1+ license as defined in our [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

----
📚 Documentation preview 📚: https://mdanalysis--4809.org.readthedocs.build/en/4809/

### CODE REVIEW

File: testsuite/MDAnalysisTests/transformations/test_rotate.py [hunk 1]
```diff
@@ -71,17 +77,18 @@ def test_rotateby_custom_point(rotate_universes, point):
     trans = trans_u.trajectory.ts
     ref = ref_u.trajectory.ts
     vector = [1, 0, 0]
-    pos = point.reshape(3, )
+    pos = point.reshape(
+        3,
+    )
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/transformations/test_rotate.py]:
```suggestion
    pos = point.reshape(3)
```
I think this is equivalent, unless I'm missing something. Unnecessary trailing commas lead to odd formatting.

File: testsuite/MDAnalysisTests/transformations/test_rotate.py [hunk 2]
```diff
@@ -91,7 +98,9 @@ def test_rotateby_vector(rotate_universes, vector):
     ref = ref_u.trajectory.ts
     point = [0, 0, 0]
     angle = 90
-    vec = vector.reshape(3, )
+    vec = vector.reshape(
+        3,
+    )
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/transformations/test_rotate.py]: See above.

File: testsuite/MDAnalysisTests/transformations/test_rotate.py [hunk 3]
```diff
@@ -45,24 +48,27 @@ def test_rotation_matrix():
     angle = 180
     vector = [0, 0, 1]
     pos = [0, 0, 0]
-    ref_matrix = np.asarray([[-1, 0, 0],
-                             [0, -1, 0],
-                             [0, 0, 1]], np.float64)
+    ref_matrix = np.asarray([[-1, 0, 0], [0, -1, 0], [0, 0, 1]], np.float64)
```
Comment (marinegor) [inline testsuite/MDAnalysisTests/transformations/test_rotate.py]:
```suggestion
    ref_matrix = np.asarray([[-1, 0, 0], [0, -1, 0], [0, 0, 1],], np.float64)
```

**Review by marinegor — APPROVED** [review]
LGTM except for the `.git-ignore-blame-revs` file update, which I recall you're doing at the end anyway

**Comment by RMeli** [comment]
> which I recall you're doing at the end anyway

Thanks @marinegor for the review! Yes, I'll update the file in a separate PR, pooling some of the `[fmt]` PRs, the reason being that cleaning up history to only have two commits is just too tedious.

File: package/MDAnalysis/transformations/rotate.py [hunk 4]
```diff
@@ -144,38 +148,47 @@ def __init__(self,
...
-            if self.direction.shape != (3, ) and \
-               self.direction.shape != (1, 3):
-                raise ValueError('{} is not a valid direction'
-                                 .format(self.direction))
-            self.direction = self.direction.reshape(3, )
+            if self.direction.shape != (3,) and self.direction.shape != (1, 3):
+                raise ValueError(
+                    "{} is not a valid direction".format(self.direction)
+                )
+            self.direction = self.direction.reshape(
```
Comment (IAlibay) [inline package/MDAnalysis/transformations/rotate.py:155]: I'm not going to block on this.. but surely this is a single line thing?

**Review by IAlibay — APPROVED** [review]

File: package/MDAnalysis/transformations/rotate.py (hunk 4 above)
Comment (RMeli) [reply package/MDAnalysis/transformations/rotate.py:155]: Nope. With the indentation this is 86 characters long. In https://github.com/MDAnalysis/mdanalysis/pull/4717 we decided to stick with 79 characters, hence the formatting. I'd be happy to reconsider the line length, but I think that is something to do once the per-module formatting is over (if there is consensus).

**Pull request merged**